net-irc/znc/znc-1.6.6.ebuild

```diff
@@ -0,0 +1,130 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{4,5,6} )
+inherit eutils python-single-r1 readme.gentoo-r1 systemd user
+
+MY_PV=${PV/_/-}
+GTEST_VER="1.7.0"
+GTEST_URL="https://github.com/google/googletest/archive/release-${GTEST_VER}.tar.gz -> googletest-release-${GTEST_VER}.tar.gz"
+DESCRIPTION="An advanced IRC Bouncer"
```

Comment:
Could you please reorder the variables to match the skel? It would make reading much easier

```
DESCRIPTION
HOMEPAGE
SRC_URI

LICENSE
SLOT
KEYWORDS
IUSE
REQUIRED_USE
```

Since you have 3 extra vars on top, please also add an empty line between them and first metadata vars.